add demographic dimensions (#34)

GeorgLink opened the pull request:
adding:
* Romantic Orientation
* Religion

GeorgLink commented:
fixes #34

Erioldoesdesign commented on demographic-data/README.md:
```diff
@@ -25,17 +25,19 @@ NOTE: that *how* we might ask these questions, and what potential answers might
 2. Sexual orientation
-3. Age
-4. Location/Region/Country
+3. Romatic orientation
```
Spell check on 'Romantic'

GeorgLink replied:
Thanks. I fixed it.

GeorgLink commented:
I changed them to have all "1." in front of them because then a commit would only need to edit one line if we added a characteristic in the middle of the list. The problem with numbering them subsequentially is that inserting an entry requires changing all the following lines and it muddies the commit history.

Having all entries with "1." is proper Markdown formatting.

germonprez commented:
Cool! Thanks for the detail @GeorgLink
